conf.yaml

```diff
@@ -525,6 +526,21 @@ contents:
                 path:   docs/shared
                 exclude_branches: [ 1.0 ]
 
+         -
+            title:      Elastic Cloud on Kubernetes
+            prefix:     en/k8s
```

Comment:
Can we make this prefix match the repo name? `cloud-on-k8s`

Reply:
Done.